test.js

```diff
@@ -75,9 +84,8 @@ describe('v8flags', function () {
 
   it('should fall back to writing to a temp dir if user home is unwriteable', function (done) {
     eraseHome();
-    env.HOME = path.join(__dirname, 'does-not-exist');
+    env.HOME = env.LOCALAPPDATA = path.join(__dirname, 'does-not-exist');
     const v8flags = require('./');
-    v8flags.configPath = env.HOME;
```

Comment:
I assume this is a leftover from before I changed them. Doesn't look needed any more.